common/metadata.go

```diff
@@ -39,6 +40,35 @@ type BridgingRequestMetadata struct {
 	Transactions       []BridgingRequestMetadataTransaction `cbor:"tx" json:"tx"`
 	FeeAmount          uint64                               `cbor:"fa" json:"fa"`
 }
+*/
+
+type BridgingRequestMetadataAmount struct {
+	SourceCurrencyAmount      uint64 `cbor:"sca" json:"sca"`
+	DestinationCurrencyAmount uint64 `cbor:"dca" json:"dca"`
+}
+
+// IsNativeToken - is the user trying to bridge native tokens (WAda, WApex), or native currency (Ada, APEX)
+// Additional will be counted towards the bridging fee shown to the user, but it will actually be assigned to
+// the user on destination chain, because we can not create a UTXO with only native tokens
+// Additional will be nil for reactor
+// Additional.DestinationCurrencyAmount is minUtxoAmount on destination chain
+// Additional.SourceCurrencyAmount is Additional.DestinationCurrencyAmount * exchangeRate
+type BridgingRequestMetadataTransaction struct {
+	Address       []string                       `cbor:"a" json:"a"`
+	IsNativeToken bool                           `cbor:"nt" json:"nt"`
+	Amount        uint64                         `cbor:"m" json:"m"`
+	Additional    *BridgingRequestMetadataAmount `cbor:"ad" json:"ad"`
```

Comment:
maybe we should have made this not a pointer.. so we don't have to constantly check if .Additional != nil

Reply:
Its better to stay pointer because if not, `Additional` will be always included in metadata. Just add `omitempty` tag

Reply:
yes, it will, but we wouldn't have if .Additional != nil everywhere

Reply:
or, we could leave it as pointer, and do:
 if receiver.Additional == nil { reciever.Additional = Tokens{}
}

at the top of the loop